### PR TITLE
Add support for Puppet 5

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

**Comment by Phil-Friderici** [comment]
Travis is active on my repository. You can check the result here: https://github.com/Phil-Friderici/puppet-module-env/pull/1

File: manifests/path.pp [hunk 1]
```diff
@@ -12,7 +12,7 @@
   $directories           = 'MANDATORY',
 ) {
 
-  include env
+  include ::env
```
Comment (ghost) [inline manifests/path.pp:15]: Hi Phil,

Why the double colon in the include? I didn't find any reason for it in the documentation.
https://puppet.com/docs/puppet/5.3/lang_classes.html#using-include
https://puppet.com/docs/puppet/5.5/lang_namespaces.html

Thanks

Comment (Phil-Friderici) [reply manifests/path.pp:15]: for more details please check https://docs.puppet.com/puppet/3/lang_namespaces.html#relative-name-lookup-and-incorrect-name-resolution

Can be removed again once Puppet 3 support is dropped.
https://github.com/voxpupuli/puppet-lint-absolute_classname-check

Comment (ghost) [reply manifests/path.pp:15]: ok good

Comment (ghost) [reply manifests/path.pp:15]: the PR's message is misleading!

**Pull request merged**